### ISSUE TITLE
nice tools but waste search

### ISSUE THREAD

**Issue by unknownleakersource** [hackthedev/Botnet#1]
nice tools but waste time because google will block ip bot search

**Comment by hackthedev** [comment]
yeah google might block it but if i have enough time i could embed a proxy client. i once made one [here](https://github.com/hackthedev/proxy-changer). shouldnt be that hard to add actually.

i need to keep updating that search tool its not done yet sadly
